api/v1/storagecluster_types.go

```diff
@@ -254,6 +254,7 @@ type ManageCephRBDMirror struct {
 type MgrSpec struct {
 	// EnableActivePassive can be set as true to deploy 2 ceph manager pods, one active and one standby
 	// Ceph will promote the standby mgr when the active mgr goes down due to any reason
+	// +kubebuilder:deprecatedversion:warning="This field has been deprecated and will be removed in future versions. Active passive mgr is now enabled by default."
 	EnableActivePassive bool `json:"enableActivePassive,omitempty"`
```

Comment:
How can the user disable this feature? While we default to 2 mgrs, it still seems valid to allow 1 mgr. Perhaps we add a new setting that is `disableActivePassive`. We still should allow 1 mgr for minimal deployments. For example, in the new profiles, the smallest profile could have a single mgr.